docs/common/common_examples.rst

```diff
@@ -755,3 +755,194 @@ EN 6. Authorization Request
       },
       "state": "fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd"
   }
+
+.. _Esempio_EN7:
+
+EN 7. Metadata Policy
++++++++++++++++++++++
+
+The following example shows a metadata policy in the Entity Statement provided by a TA and related to an RP
+
+.. code-block:: python
+
+    "metadata_policy": {
+        "openid_relying_party": {
+            "jwks": {
+                "one_of": [{
+                    "kty": "RSA",
+                    "use": "sig",
+                    "n": "…",
+                    "e": "AQAB",
+                    "kid": "5NNNoFS3YnC9tjiCaivhWLVUJ3AxwGGz_98uRFaqMEEs"
+                }]
+            }
+            "grant_types": {
+                "one_of": ["authorization_code", "refresh_token"]
```

Comment:
not one_of but subset_of, do you agree?

a leaf may want/support both auth code and refresh

Reply:
done